tests/scenario/test_ingress_per_app.py

```diff
@@ -0,0 +1,173 @@
+# GIVEN a charm with ingress impl'd
+# WHEN a relation with traefik is formed
+# THEN traefik's config file's `server` section has all the units listed
+# AND WHEN the charm rescales
+# THEN the traefik config file is updated
+
+import tempfile
+from pathlib import Path
+
+import pytest
+import yaml
+from ops import pebble
+from scenario import Container, Context, Model, Mount, Relation, State
+
+
+@pytest.fixture
+def context(traefik_charm):
+    return Context(charm_type=traefik_charm)
+
+
+@pytest.fixture
+def model():
+    return Model(name="test-model")
+
+
+@pytest.fixture
+def temp_opt():
```

Comment:
Seems like this functionality is [built-in](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html).

Reply:
yeah but pytest-operator breaks it 
![image](https://github.com/canonical/traefik-k8s-operator/assets/6230162/1a252305-788b-44e6-987a-127ea68cd238)

however it does work when running it from tox as it does not install pytest-operator.